test/e2e/node/kubelet.go

```diff
@@ -466,6 +469,9 @@ var _ = SIGDescribe("kubelet", func() {
 			numNodes  int
 			nodeNames sets.String
 		)
+		lines := func(out string) []string {
```

Comment:
not related to this PR, but can you please update a feature tag on the test to something without spaces. Like `[Feature:NodeLogViewer]` to match the style of other names. Extra bonus for using the feature gate name with this tag